UI/GameSettingsScreen.cpp

```diff
@@ -449,6 +449,10 @@ void GameSettingsScreen::CreateViews() {
 	});
 
 	CheckBox *texBackoff = graphicsSettings->Add(new CheckBox(&g_Config.bTextureBackoffCache, gr->T("Lazy texture caching", "Lazy texture caching (speedup)")));
+	texBackoff->OnClick.Add([=](EventParams& e) {
+		settingInfo_->Show(gr->T("Lazy texture caching Tip", "When enable,game font may have problem"), e.v);
```

Comment:
This might be a cosmetic, but change this to
`When enabled, game font may have problem`